helm/dpga-compliance-barebone/operations-web/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```

Comment:
duplicate files

Reply:
[PHEE-456 Remove duplicity from dpga-compliance-barebone helm chart in env labs](https://fynarfin.atlassian.net/browse/PHEE-456?atlOrigin=eyJpIjoiNjliNDVkMmM5YTQzNDE5YzkwY2U3N2QzYzE2Y2NlNDgiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)